lib/GoalCohort.js

```diff
@@ -325,18 +379,20 @@ GoalCohort.statics.createTeam = async function(user, goal, teamname) {
       );
     }
     teamIndex = cohort.teams.length;
-    teamIcon = DEFAULT_COHORT_TEAMS[cohort.teams.length].icon;
+    teamIcon = DEFAULT_COHORT_TEAMS[cohort.teams.length % 6].icon;
```

Comment:
why the `% 6`?

Reply:
Because DEFAULT_COHORT_TEAMS has 6 default teams. If you are adding a 7th team onto a named cohort, this would give an out of bounds error.

Reply:
ok then should it be `DEFAULT_COHORT_TEAMS[cohort.teams.length % DEFAULT_COHORT_TEAMS.length].icon`?